### PR TITLE
Show latest real members count using Slack API.

### PR DESCRIPTION
**Pull request by moriokumura**
This change shows real number of members in our team and each channel.

### To run this on your local, 
- add your Slack API token to your `.env` file
- then run `rake db:migrate` and `rake slack:update` to fetch Slack API.

(see [here](https://github.com/railslink/railslink/blob/count/README.md) for full setup procedure)

### Note
- On production environment, we'll run `rake slack:update` once a day or more.
- This PR doesn't include test code yet. I'm not great test coder - how should I write tests for apps like this?
- I skipped displaying geographic distribution. It requires more complicated, well-thought implementation.

### Channels section will look like this
screen shot 2015-08-13 at 10 55 40 am

### CODE REVIEW

File: app/models/slack_api_response.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class SlackApiResponse < ActiveRecord::Base
+  
+  # Returns latest valid API response.
+  #
+  # @return [SlackApiResponse]
+  def self.latest(api_name)
+    self.where(method_name: api_name, ok: true).order(updated_at: :desc).first
```
Comment (arun057) [inline app/models/slack_api_response.rb]: maybe add a limit(1) as well here so it doesnt waste time parsing all those records?

File: README.md [hunk 2]
```diff
@@ -42,5 +45,7 @@ Then run:
 ```
 bundle
 bundle exec rake db:create
+bundle exec rake db:migrate
+bundle exec rake slack:update
```
Comment (arun057) [inline README.md]: Is there going to be some sort of cron set up to run this regularly?

Comment (nauman) [inline README.md]: @arun057 isn't that going to happen only once? if it is then we can skip cron for now

Comment (arun057) [inline README.md]: I would assume you would want to run this multiple times ( once a day? or every time a new user is added? )

Comment (moriokumura) [inline README.md]: I will set Heroku Scheduler daily. This is the standard way to do cron-like task on Heroku.
https://elements.heroku.com/addons/scheduler

File: app/models/slack_api_response.rb (hunk 1 above)
Comment (moriokumura) [inline app/models/slack_api_response.rb]: When you use `first`, ActiveRecord adds `LIMIT 1` to the query.

**Pull request merged**